### ISSUE TITLE
AttributeError: module 'octohatrack' has no attribute 'main'

### ISSUE THREAD

**Issue by stewart-ibm** [LABHR/octohatrack#92]
With e1b52cddf10cedc9f1bca59db2df9f016911a8c2, 

```
# docker build -t octohatrack .
.....
# docker run -e GITHUB_TOKEN octohatrack
Traceback (most recent call last):
  File "octohatrack.py", line 2, in <module>
    octohatrack.main()
AttributeError: module 'octohatrack' has no attribute 'main'
```

**Comment by glasnt** [comment]
Hi @stewart-ibm 

Nice catch! It looks like the Dockerfile is out of date. Can you try the following change locally to see if it fixes things?

```diff
diff --git a/Dockerfile b/Dockerfile
index 880cd11..4b262ac 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -8,6 +8,6 @@ WORKDIR /usr/src/app
 COPY . /usr/src/app
 RUN pip install octohatrack

-ENTRYPOINT ["python", "octohatrack.py"]
+ENTRYPOINT ["octohatrack"]
 CMD ["-h"]
```

If it does, I'll update the Dockerfile to suit. Thanks!

**Comment by stewart-ibm** [comment]
Katie McLaughlin <notifications@github.com> writes:
> Hi @stewart-ibm 
>
> Nice catch! It looks like the Dockerfile is out of date. Can you try the following change locally to see if it fixes things?
>
> ```diff
> diff --git a/Dockerfile b/Dockerfile
> index 880cd11..4b262ac 100644
> --- a/Dockerfile
> +++ b/Dockerfile
> @@ -8,6 +8,6 @@ WORKDIR /usr/src/app
>  COPY . /usr/src/app
>  RUN pip install octohatrack
>
> -ENTRYPOINT ["python", "octohatrack.py"]
> +ENTRYPOINT ["octohatrack"]
>  CMD ["-h"]
> ```
>
> If it does, I'll update the Dockerfile to suit. Thanks!

It does work, thanks!

-- 
Stewart Smith
OPAL Architect, IBM.

**Comment by glasnt** [comment]
Awesome! This is being fixed over in #93
